src/pages/EWFormPage/index.js

```diff
@@ -94,6 +101,49 @@ const referrals = [
   'Other',
 ];
 
+function getWindowDimensions() {
+  const { innerWidth: width, innerHeight: height } = window;
+  return {
+    width,
+    height
+  };
+}
+
+const inputMobileStyle = {
+  width: "74vw",
+  color: "#8CC9BA",
+  borderRadius: 10,
+  border: "2px solid #8CC9BA",
+  backgroundColor: "#FFF",
+  height: "14vh",
+  fontSize: "12px",
+  fontFamily: "Montserrat",
+}
+
+const inputMobileStyle1 = {
+  width: "74vw",
+  color: "#8CC9BA",
+  borderRadius: 10,
+  border: "2px solid #8CC9BA",
+  backgroundColor: "#FFF",
+  height: "19vh",
+  fontSize: "11.5px",
```

Comment:
again, too small

Reply:
I agree 16px size is great for desktop, but on mobile, due to the limited screen space, I wanted to compact things so as to fit all the components in a reasonable space proportionality 

This is what it looks like when the font size is larger (placeholders cut off at the bottom and textarea would need to be made a lot larger if the font was bigger), let me know your thoughts: 
<img width="297" alt="Screen Shot 2020-09-04 at 10 52 58 PM" src="https://user-images.githubusercontent.com/40772490/92296368-f30de000-ef01-11ea-971e-1c903080b3fe.png">
<img width="279" alt="Screen Shot 2020-09-04 at 10 53 27 PM" src="https://user-images.githubusercontent.com/40772490/92296371-f3a67680-ef01-11ea-97a3-16b066637bc8.png">